pulp_python/app/modelresource.py

```diff
@@ -0,0 +1,15 @@
+from pulpcore.plugin.importexport import BaseContentResource
+from pulp_python.app.models import (
+    PythonPackageContent,
+)
+
+class PythonPackageContentResource(BaseContentResource):
+    """
+    Resource for import/export of python_pythonpackagecontent entities.
+    """
```

Comment:
The link above is now:

https://docs.pulpproject.org/pulpcore/plugin_dev/plugin-writer/concepts/subclassing/import-export.html

`set_up_queryset()` is The Thing that tells the import/export machinery how to find all the entities associated with a specific RepositoryVersion, and hence what to export. In this specific model-resource, it would a filter that would return all `PythonPackageContent` in a specified repo-version.